js/src/widgets/metadataView.js

```diff
@@ -164,11 +163,7 @@
     var label = attr;
 
     label = label.replace(/^@/, '');
-    label = label.replace(/([A-Z])/g, ' $1');
     label = label.replace(/\s{2,}/g, ' ');
-    label = label.replace(/\w\S*/g, function(txt) {
```

Comment:
@jbaiter Agreed that this should be a user choice, and otherwise falls on the repository or publisher of the manifest. I will merge this as is, and then do another PR to remove the CSS capitalisation.